packages/playground/src/components/select_node.vue

```diff
@@ -188,43 +212,115 @@ watch(
       value.certified === oldValue.certified &&
       value.rentedBy === oldValue.rentedBy &&
       value.hasGPU === oldValue.hasGPU
     )
       return;
-    shouldBeUpdated.value = true;
+    if (props.selection === "manual" && selectedNode.value) {
+      await validateManualSelectedNode(selectedNode.value);
+    } else {
+      shouldBeUpdated.value = true;
+    }
+  },
+  { deep: true },
+);
+
+watch(
+  () => props.selection,
+  async (value, oldValue) => {
+    errorMessage.value = ``;
+    if (value !== oldValue) {
+      selectedNode.value = undefined;
+      ManualselectedNode.value = undefined;
+    }
   },
   { deep: false },
 );
 
+watch(
+  () => ManualselectedNode.value,
+  async (value, oldValue) => {
+    if (value != undefined || value != null) {
+      selectedNode.value = { nodeId: Number(value) };
+    }
+  },
+  { deep: false },
+);
 watch([loadingNodes, shouldBeUpdated], async ([l, s]) => {
   if (l || !s) return;
   shouldBeUpdated.value = false;
-  farmManager?.subscribe(farmId => {
-    if (!farmId) {
-      selectedNode.value = undefined;
-      availableNodes.value = [];
-      return;
-    }
-    clearTimeout(delay.value);
-    delay.value = setTimeout(() => {
-      loadNodes(farmId);
-    }, 1000);
-  });
+  if (props.selection === "automated") {
+    farmManager?.subscribe(farmId => {
+      if (!farmId) {
+        selectedNode.value = undefined;
+        availableNodes.value = [];
+        return;
+      }
+      clearTimeout(delay.value);
+      delay.value = setTimeout(() => {
+        loadNodes(farmId);
+      }, 1000);
+    });
+  }
 });
 
 function getChipColor(item: any) {
   return item === "Dedicated" ? "success" : item === "Certified" ? "primary" : "secondary";
 }
-async function loadNodes(farmId: number) {
+function validateSelectedNodeFilters(
+  validatingNode: INode,
+  node: NodeInfo,
+  freeresources: NodeResources | undefined,
+  filters: NodeFilters,
+) {
+  if (node.inDedicatedFarm && node.rentedByTwinId !== profileManager.profile?.twinId) {
+    errorMessage.value = `Node ${validatingNode.nodeId} belongs to a dedicated farm and not rented by you`;
+  } else if (node.rentedByTwinId !== profileManager.profile?.twinId && node.rentedByTwinId !== 0) {
+    errorMessage.value = `Node ${validatingNode.nodeId} is rented by someone else`;
+  } else if (filters.ipv4 && !node.publicConfig.ipv4) {
+    errorMessage.value = `Node ${validatingNode.nodeId} is not assigned to a public ip`;
+  } else if (freeresources && freeresources.cru < filters.cpu) {
+    errorMessage.value = `Node ${validatingNode.nodeId} doesn't have enough CPU`;
+  } else if (freeresources && freeresources.mru < Math.round(filters.memory / 1024)) {
+    errorMessage.value = `Node ${validatingNode.nodeId} doesn't have enough memory`;
+  } else if (freeresources && freeresources.sru < filters.diskSizes.reduce((total, disk) => total + disk)) {
+    errorMessage.value = `Node ${validatingNode.nodeId} doesn't have enough SRU`;
+  }
+}
```

Comment:
- Can we remove this to a separate file and export it to use it here instead?  
- We can use optional chaining so instead of doing 
freeresources && freeresources.cru, we can do freeresources?.cru. This goes for all conditions. 
- Maybe divide validations into smaller functions and check them all in here?